h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/XGBoostMojoModel.java

```diff
@@ -19,12 +19,13 @@
   public boolean _useAllFactorLevels;
   public boolean _sparse;
 
-  public XGBoostMojoModel(String[] columns, String[][] domains) {
-    super(columns, domains);
+  public XGBoostMojoModel(String[] columns, String[][] domains, String responseColumn) {
```

Comment:
Theoretically, this is breaking change in API, because of the change in public ctor signature.

Reply:
In theory yes, however I don't see why would anyone be instantiating the MojoModel directly and not using MojoReader.

I would assume that this could happen if someone was building their MOJOs for their own algos.